wordpress-mobile/WordPress-Android
# Use vector drawable for the insights management news card

Fixes #11723

This PR replaces the PNG drawables used for the stats management news card with a vector drawable partially covered with a gradient. 

![Screenshot_1592833163](https://user-images.githubusercontent.com/1079756/85296138-37f8ad80-b4a1-11ea-9c7e-46b6cf3f2df5.png)
![Screenshot_1592833148](https://user-images.githubusercontent.com/1079756/85296146-3a5b0780-b4a1-11ea-8ca4-d5d5a1522bde.png)


To test:
- Go to stats (on a site where you haven't used insights management before)
- Check that there is the updated news card

PR submission checklist:
